projects/storefrontlib/src/lib/pwa/pwa.module.ts

```diff
@@ -28,7 +34,14 @@ export function pwaConfigurationFactory(
       provide: RegistrationOptions,
       useFactory: pwaConfigurationFactory,
       deps: [Config]
-    }
+    },
```

Comment:
why not just call "init" in the service's constructor. So, this part can be removed.

Reply:
Removed `init` call after service refactor.